Monday August 19th 2013 7:20 pm -04:00,I am using my husbands Mac OS Snow Leopard Version 10.68 and I dont know which Idle to use. How do you tell which one you should use? Thanks, Resolved,1
Monday August 19th 2013 7:26 pm -04:00,I just found an answer that I will post in case someone has the same problem. I have a 64 bit Mac. Here is a good link. http://apple.stackexchange.com/questions/23713/how-do-i-figure-out-if-my-os-is-mac-os-x-32-bit-or-64-bit , Resolved,1
Friday August 23rd 2013 11:16 pm -04:00,Hi I have a window with a black background and some text in Windows vista. I cannot find RUN either. It says that it is in IDLE.There is no FILE.  . Everything else seems to be working like the calculator and I can get the builtin menu and call up the help functions but there is nothing else. What do I do?. Thanks. , Resolved,1
Saturday August 24th 2013 12:14 am -04:00,Wow thank you so much! I think I have it now. : ), Resolved,0
Thursday September 5th 2013 10:28 pm -04:00,I got this question wrong all five times. I hope they will tell us the right answer. , Resolved,0
Tuesday August 27th 2013 5:08 am -04:00,I have not attempted this weeks assignment yet however I notice that assignment one shows a score. Im not sure if this is a bug or what happened. It looks like this. PartNameLast SubmissionScoreFeedback1 / 1Assignment1-- / 41ViewSubmitTotal Score0 / 41, Resolved,0
Tuesday August 27th 2013 2:49 pm -04:00,Ok I see. I just wanted to make sure that it hadnt been posted as blank some how. The exercises are different they show 0 of 5 so none have been tried. Is this right? , Resolved,0
Tuesday August 27th 2013 2:55 pm -04:00,Mine says this: Part 1	Feedback	1 / 1	Assignment1	-   - / 41	 Total Score		0 / 41          , Resolved,0
Thursday September 5th 2013 10:30 pm -04:00,I hope they will give us the right answer on this one. I got it wrong five times and have no clue about the right answer. , Resolved,0
Saturday August 31st 2013 11:38 am -04:00,I keep getting an error that Canada is not defined. Are we to use the code exactly as it is? (This is true for Question 5 as well), Unresolved,0
Saturday August 31st 2013 4:34 pm -04:00,I appreciate your answer even though I still cant figure half of these out. Ive been at them for 2 whole nights. I have to give the computer back. Oh well I did my best. , Unresolved,0
Sunday September 1st 2013 11:42 am -04:00,I had been using my husbands apple computer when I started to submit this assignment. He needed it back so I am now on my windowss vista PC. After I submitted my last two trys the grader is not  releasing them. I was wondering if using two different computer systems would be a problem. Thanks for your help. (I had to redo the code for the vista --I just need to know if I have to run it through the Mac again in which case I will have to go to my fathers house and use a third computer and do it all over again. )Raw ScoreNot available, Resolved Locked,0
Wednesday September 4th 2013 9:04 pm -04:00,Are you at the assignment page?, Unresolved,0
Wednesday September 4th 2013 9:08 pm -04:00,Step 7: Submit your workGo to the Assignments page and click the appropriate Submit button. Choose your completed a1.py file. It should be marked within a few minutes. You can view your results by clicking on the View button in the Feedback column., Unresolved,0
Monday September 9th 2013 7:35 am -04:00,Im still getting an error. I cant run this in the python shell. , Resolved,-2
Monday September 9th 2013 9:19 am -04:00,Nevermind. I can figure it out without running it but I would like to know how to do it with Python on a PC or a Mac., Resolved,0
Monday September 9th 2013 10:22 am -04:00,I am on a PC now and I am getting the same error as Andris above. I have tried defining x but that doesnt help in running the other suggestions. Since the question suggests we try this in the shell it must be possible.3 <= int(x) < 10Traceback (most recent call last):  File <pyshell#0> line 1 in <module>    3 <= int(x) < 10NameError: name x is not definedI used the built in functions to get the int(x) functions and I tried to define it with bool. I do hope someone will go over the homework and in lesson quizes at some point. , Resolved,0
Monday September 9th 2013 12:29 pm -04:00,Thanks I finally got it in Python. , Resolved,0
Monday September 9th 2013 12:59 pm -04:00,Is that true? Because then I must have a bug in my python  or I just cant see it. , Unresolved,0
Wednesday September 18th 2013 6:11 am -04:00,I have tried for 2 nights to get this question to run and have several run time errors. I assume it must be simple but I am trying to change something that is a True statement to say False which Python doesnt seem to want to do. I can logically figure out this question without Python but would like to be able to do it in the shell. I have tried every suggestion here and others. Is it really possible to get True to say False? Thanks, Unresolved,-1
Friday September 20th 2013 2:59 pm -04:00,I cant find them Alain. Where would they be? Thanks. , Resolved,0
Friday September 20th 2013 11:18 pm -04:00,Thanks Alain...and I just noticed the python code is posted there.  That should help a lot. Thanks so much. , Resolved,0
Tuesday September 10th 2013 8:26 am -04:00,I know the feeling. I am spending far to much time on this. It is taking over my life. It did help to start over at the beginning. What do you have so far Harmony? , Resolved,3
Tuesday September 10th 2013 8:38 am -04:00,Have you turned in what you have? When I started over I got a few points little by little. Also it showed where I went wrong. I actually started over 5 times. You could take a break (very important) and give it another try. , Resolved,7
Thursday September 12th 2013 1:51 am -04:00,I really dont understand. I have been at this question for hours. I think I will just let it go but I hate to do it., Resolved,0
Thursday September 12th 2013 2:50 am -04:00,I understand having to look for the first Sandy at 6 but are you saying that I look for Sandy at 7 and 8 too. I just dont think shell be there. , Resolved,0
Thursday September 12th 2013 3:06 am -04:00,Thanks Tom. I appreciate your help. Have a good night., Resolved,0
Saturday September 14th 2013 1:58 pm -04:00,See what happens if you take away the plus sign and replace it with a comma. Your second function looks correct. (You just need to fill it in with the right parenthesis) something1.find(your correct function) with your correct parenthesis. , Resolved,0
Saturday September 14th 2013 2:00 pm -04:00,Let me know if this is wrong. I am just a beginner too. I will take it off. , Resolved,0
Saturday September 14th 2013 2:07 pm -04:00,Thanks Tom, Resolved,0
Sunday September 15th 2013 5:45 am -04:00,Take away your quotes    and add a (  parenthesis after the first .find(   then match the parenthesis on the end ).I dont think I had a space after the comma s1.find(second_parameter).  Somehow it worked for me (I hope I am explaining it right). , Resolved,0
Sunday September 15th 2013 6:51 am -04:00,Try the code with a illegal capital letter and see what happens.Looks like you need to put in an example for a lowercase letter also. They give you two choices in the assignment PDF. One could be to restrict every illegal possiblility. Thats what I did and it worked.(it was a pretty long list.) , Resolved,0
Monday September 16th 2013 1:37 am -04:00,I have been trying to figure out what I did wrong on question 15 Test 2 about the capitals. Finally the answer on my 7th try was marked correct. When I checked back on my test 5 I had the same answer. Is it possible the autograder could be wrong? Could you please look into this. Thank you very much. , Resolved,1
Monday September 16th 2013 2:09 am -04:00,Well it sure did drive me crazy. So it just stays wrong then? Too bad. Thanks, Resolved,0
Monday September 16th 2013 2:37 am -04:00,Yes thanks that was a great idea and it taught me a lot. (They did match by the way.) Thank you both for your replies., Resolved,1
Monday September 16th 2013 2:49 am -04:00,Wait I just tested it again and I found a typo. Sorry about that. It was hard to see., Resolved,1
Wednesday September 18th 2013 8:43 pm -04:00,In my get_answer program it is only returning two questions and then giving the final one as an answer even if it isnt yes or no. What am I doing wrong? def get_answer(prompt):    (str) -> str    Use prompt to ask the user for a yes or no    answer and continue asking until the user gives    a valid respones. Return the answer.        answer = input(prompt)    while not (answer == yes or answer == no):        answer = input(prompt)        return answerHere are some of my examples get_answer(Are you tired? Answer yes or no)Are you tired? Answer yes or no maybeAre you tired? Answer yes or no quite quite>>> get_answer(Is it late?)Is it late? siIs it late?yesyes>>> ================================ RESTART ================================>>> >>> get_answer(Are you tired? Answer yes or no)Are you tired? Answer yes or noAre you tired? Answer yes or no, Resolved,0
Wednesday September 18th 2013 9:22 pm -04:00,Wow Thanks a lot Tom. Now it works!, Resolved,0
Wednesday September 18th 2013 9:34 pm -04:00,Whoops its not stopping when I answer Yes or no. Any suggestions?def get_answer(prompt):    (str) -> str    Use prompt to ask the user for a yes or no    answer and continue asking until the user gives    a valid response. Return the answer.        answer = input(prompt)    while not (answer == yes or answer == no):        answer = input(prompt)    return answer>>> get_answer(Are you tired? Answer yes or no)Are you tired? Answer yes or no quiteAre you tired? Answer yes or no give me fiveAre you tired? Answer yes or no Ill sayAre you tired? Answer yes or no  reallyAre you tired? Answer yes or no Are you tired? Answer yes or noAre you tired? Answer yes or noAre you tired? Answer yes or noAre you tired? Answer yes or no yer or noAre you tired? Answer yes or noyes or noAre you tired? Answer yes or no noAre you tired? Answer yes or no noAre you tired? Answer yes or no, Resolved,0
Thursday September 19th 2013 12:33 am -04:00,Thanks. for your help. , Resolved,0
Thursday September 19th 2013 7:46 pm -04:00,I had to make sure I started from a restarted shell. This is what I did. grades = [60 70]>>> grades.append(80)>>> grades[60 70 80]>>> grades.append(90)>>> >>> grades[60 70 80 90]>>> , Unresolved,1
Sunday September 22nd 2013 7:11 pm -04:00,I too am having problems running this code in idle. Here is how I set it up I used my husbands mac but now I have to go back to the vista. Here are some of the examples. Did I set it up right? (I have used both square and round brackets). def cap_song_repetition(playlist):    (list of str str) -> list of string   playlist = (A B A A C A D E A A)    cap_song_repetition(playlist A)    print(playlist)    Make sure there are no more than 3 occurrences of song in playlist.    playlist = [A B A A C A D E A A]    cap_song_repetition(playlist A)    print(playlist)    while playlist.count(song) >= 3:            playlist.remove(song)               ##                ##    while playlist.count(song) > 3:##        playlist.remove(song)##playlist = [A B A A C A D E A A]##cap_song_repetition(playlist A)##print(playlist)##     while playlist.count(song) > 3:##        playlist.remove(playlist.index(song))##	####    while playlist.count(song) >= 3:##        playlist.remove(song)######    while playlist.count(song) > 3:##        playlist.remove(song)##	 Some errors that i have received>>> print(playlist)Traceback (most recent call last):  File <pyshell#11> line 1 in     print(playlist)NameError: name playlist is not defined>>> NameError: name playlist is not definedSyntaxError: invalid syntax>>> >>> >>> >>>     playlist = [A B A A C A D E A A]    cap_song_repetition(playlist A)    print(playlist)    SyntaxError: unexpected indent>>> >>> playlist = [A B A A C A D E A A]    cap_song_repetition(playlist A)print(playlist)SyntaxError: multiple statements found while compiling a single statemen, Resolved,0
Sunday September 22nd 2013 7:18 pm -04:00,I am trying to unresolve this thread, Resolved,0
Sunday September 22nd 2013 9:07 pm -04:00,Thanks so much for responding.. Are you saying we dont need to run this in Python? I just did it on the windows and came up with a name error that playlist is not defined. def cap_song_repetition(playlist):    (list of str str) -> list of string   playlist = (A B A A C A D E A A)     cap_song_repetition(playlist A)    print(playlist)    Make sure there are no more than 3 occurrences of song in playlist. >>> cap_song_repition = [(A Lola)( BVenus)(ALola)(A Lola) (CLet it be) (D ABC)(E Cecelia) (A Lola)         >>>(ALola)] >>> cap_song_repetition(playlist A]>>> print(playlist)##playlist = [Lola Venus Lola Lola Let It Be Lola ABC Cecilia Lola Lola] - (song)##(song) = Lola##cap_song_repetition(playlist song)print(playlist)while playlist.count(song) > 3:        playlist.remove(song)playlist = [Lola Venus Lola Lola Let It Be Lola ABC Cecilia Lola Lola]##>>> playlist = [A B A A C A D E A A##playlist = [Lola Venus Lola Lola Let It Be Lola ABC Cecilia Lola Lola] - (song)##(song) = Lola##cap_song_repetition(playlist song)Im getting a name error now.ameError: name playlist is not defined>>> ================================ RESTART ================================>>> Traceback (most recent call last):  File C:\Python32\playlist.py line 15 in <module>    print(playlist)NameError: name playlist is not defined>>> ================================ RESTART ================================>>> Traceback (most recent call last):  File C:\Python32\playlist.py line 15 in     print(playlist)NameError: name playlist is not defined>>> , Resolved,0
Sunday September 22nd 2013 10:10 pm -04:00,SyntaxError: invalid syntaxWell I have tried to define playlist every way possible. I cant think of any other way. If anyone has any other ideas I would really appreciate it. Frankly I would not mind hearing ::Lola that many times. >>> def playlist [Lola Venus Lola Lola Let It Be Lola ABC Cecilia Lola Lola]:SyntaxError: invalid syntax>>> def playslist (Lola Venus Lola Lola Let It Be Lola ABC Cecilia Lola Lola):	SyntaxError: invalid syntax>>> def playlist Lola Venus Lolo Let it Be Lola ABC Cecilia Lola Lola Lola:SyntaxError: invalid syntax>>> def playlist (Lola Venus Lola Let it Be Lola ABC Cecilia Lola Lola Lola):    SyntaxError: invalid syntax>>>def playlist [Lola Venus Lola Lola Let It Be Lola ABC Cecilia Lola Lola]SyntaxError: invalid syntax, Resolved,0
Sunday September 22nd 2013 10:44 pm -04:00,Thank you for your answer Susan. So I am trying that now. Do I need to define it. I am so confused. , Resolved,0
Sunday September 22nd 2013 10:51 pm -04:00,Now Im getting this error. [stuff morestuff]Traceback (most recent call last):  File C:\Python32\playlist.py line 37 in <module>    while playlist.count(song) > 3:NameError: name , Resolved,0
Sunday September 22nd 2013 11:20 pm -04:00,Thanks I commented out the ones I was not using. I thought that would work. The visualizer is giving me a syntax error with the strings. I have tried one solution at a time. I have been doing this problem for 2 days straight. I dont think I have gained enough knowledge to complete it. I can give a good guess but not from the Python which does not seem right. Thanks for your help. , Resolved,0
Sunday September 22nd 2013 11:33 pm -04:00,I have to do that because I cant do it in python.  I have watched every lecture half a dozen or more times. Sometimes I can get it to work but not always. , Resolved,0
Monday September 23rd 2013 12:00 am -04:00,Thanks Ill give it a try right now. You have been very helpful Karl. , Resolved,0
Monday September 23rd 2013 1:12 am -04:00,I finally got it. Thanks so much for your help., Resolved,0
Friday September 20th 2013 4:30 am -04:00,Is it just me? My idle cant do this code because of the compress_list and compressed._list.confusion.  Is there some sort of error in the doc string? I have gotten this error 4 times. What do you think?     Traceback (most recent call last):  File <pyshell#43> line 1 in     compress_list([a b c d])  File C:/Python32/compressedlist.py line 17 in compress_list    compressed_list.append(L[i] + L[i + 1])MemoryError, Resolved,0
Sunday September 22nd 2013 5:21 pm -04:00,How do you print it out to a PDF. I cant figure that out but it sounds like a great idea. , Resolved,0
Sunday September 22nd 2013 3:39 am -04:00,Yes thats happened to me a few times. I make sure I save each answer on the quiz by clicking on save answers. It is a button on the bottom to the right of the submit button. The test wont submit until you check the box for your name.  , Resolved,1
Thursday September 19th 2013 3:10 am -04:00,I cant get any of them to work even though the correct answer is shown. Can someone please explain what Im doing wrong. Thanks.What does s refer to after this code is executed? s = names = [Alice Bob Carol]for name in names:    s = s + nameHere are some problems I had.>>> s = [Alice Bob Carol]names = [Alice Bob Carol]for name in names:    s = s + name    SyntaxError: invalid syntax>>> s = [Alice Bob Carol]names = [Alice Bob Carol]for name in names:    s = s + name    >>> >>> >>> s = AliceBobCarolnames = [Alice Bob Carol]for name in names:    s = s + name    >>> >>> >>> >>> >>> >>> s = Carolnames = [Alice Bob Carol]for name in names:    s = s + name    >>> >>> >>> >>> s = [Alice Bob Carol]names = [Alice Bob Carol]for name in names:    s = s + name    SyntaxError: invalid syntax>>> s = names = [Alice Bob Carol]for name in names:    s = s + name, Resolved,0
Thursday September 19th 2013 3:36 am -04:00,Thanks so much! I finially got it to work. , Resolved,0
Thursday September 19th 2013 3:23 pm -04:00,What if you dont have microsoft word or excell on your computer? I have a windows vista that did not come with word only notepad. Will this be a problem? ,None,0
Thursday September 19th 2013 7:00 pm -04:00,I have been over this a few times and I have the same problem. My list will not stop prompting so I cant get to the next step unless I control C and stop it. Can anyone see what Im doing wrong?>>> colours = >>> prompt = Enter another one of your favourite colours (type return to the end): >>> >>> colour = input(prompt)Enter another one of your favourite colours (type return to the end): blue>>> colourblue>>> colours>>> while colour != :	colours.append(colour)	colour = input(prompt)	Enter another one of your favourite colours (type return to the end): yellowEnter another one of your favourite colours (type return to the end): brownEnter another one of your favourite colours (type return to the end): (This is where I  returned to the end) Enter another one of your favourite colours (type return to the end):          (I kept returning to the end here)                                                                                                                  Enter another one of your favourite colours (type return to the end): I also tried it with the long vertical line after the colon and the same thing happened. Does anyone have a clue? Thanks. , Unresolved,0
Thursday September 19th 2013 8:43 pm -04:00,I copied and pasted it from my python so I have tried this half a dozen times now. I am working on everyones suggestions. Thanks so much!, Unresolved,0
Thursday September 19th 2013 8:46 pm -04:00,Now an empty string   works so that is a good start. , Unresolved,1
Friday September 20th 2013 12:14 am -04:00,My best solution right now is to do what Yeray suggested (above) and redo the program with a well placed space after my prompt phrase. But the only thing that stops it prompting is when I type the empty string  . Its not perfect but at least it stops. (I have a windows computer and I used control c to stop it running without having to close the program). I cant see where I did something wrong although it was a little hard to see (and hear) in the lecture. , Unresolved,0
Friday September 20th 2013 11:26 pm -04:00,I have already submitted my test but I would have liked to have tested this question in idle. This kept happening last night and I was wondering if it has happened to anyone else. Is it just me? My idle cant do this code because of the compress_list and compressed._list.confusion.  Is there some sort of error in the doc string? I have gotten this error 4 times. What do you think? def compress_list(L):     (list of str) -> list of str    Return a new list with adjacent pairs of string elements from L    concatenated together starting with indices 0 and 1 2 and 3    and so on.    Precondition: len(L) >= 2 and len(L) % 2 == 0    >>> compress_list([a b c d])    [ab cd]        compressed_list =     i = 0    while i < len(L):        compressed_list.append(L[i] + L[i + 1])        # MISSING CODE HERE    return compressed_listSelect the missing line of code.    Traceback (most recent call last):  File <pyshell#43> line 1 in    compress_list([a b c d])  File C:/Python32/compressedlist.py line 17 in compress_list    compressed_list.append(L[i] + L[i + 1])MemoryError, Resolved,1
Friday September 20th 2013 11:48 pm -04:00,It happened with all of them so thats why I was thinking it was in the program code or maybe the way I wrote it...., Resolved,0
Friday September 20th 2013 11:54 pm -04:00,I did it both ways. Maybe I am doing it wrong. Ill give it another try., Resolved,0
Saturday September 21st 2013 12:17 am -04:00,Its no use. Even with the right code Im getting syntax errors and return out of function calls. My idle just doesnt like this one. , Resolved,1
Saturday September 21st 2013 2:41 am -04:00,I cant get beyond step one in the visualizer. I cant even paste the link. Thanks anyway. , Resolved,0
Saturday September 21st 2013 4:15 am -04:00,In my idle they all come out the same . I hope this link to drop box works. https://www.dropbox.com/s/u5star3yjuz4xgy/numero4.txt, Resolved,0
Saturday September 21st 2013 5:34 am -04:00,Thanks it finally worked out. You were all a big help. , Resolved,0
Sunday September 22nd 2013 5:24 pm -04:00,I had trouble with that question in idle myself. I am also an old fogies student. I am going to try it again on my husbands mac computer as I think my vista has been exhausted by all the hard use it has been getting.yes_im_that_old(old_fogies (Sandy Q)  Thanks Tom and Larry, Unresolved,1
Sunday September 22nd 2013 7:24 pm -04:00,As I understand your grade is the cumulative grade so far so if you had a 13/ 24 you still  have it. Try to go back to that code and start from there. Good luck!, Unresolved,0
Wednesday September 25th 2013 7:28 pm -04:00,I have windows vista with notepad and do not have microsoft word. I dont know what to do about these documents. Do I put them into python? I have notepad ++ but that does not seem to help either. I really dont know how to procede with this. Any suggestions? , Unresolved,0
Thursday September 26th 2013 1:21 pm -04:00,Thanks Tolunay. I have been working on this for hours. I am using notepad with UTF-8. This is what I am running into. Python 3.2.3 (default Apr 11 2012 07:15:24) [MSC v.1500 32 bit (Intel)] on win32Type copyright credits or license() for more information.>>> flanders Location = (C:\Users\Sandy\Downloads\Desktop\Documents)SyntaxError: invalid syntax>>> flanders_filename = (C:\Users\Sandy\Downloads\Desktop\Documents\flanders.txt)SyntaxError: (unicode error) unicodeescape codec cant decode bytes in position 2-4: truncated \UXXXXXXXX escape>>> flanders_filename-(C://Users//Sandy//Downloads//Desktop//Documents//flanders.txt)Traceback (most recent call last):  File <pyshell#3> line 1 in <module>    flanders_filename-(C://Users//Sandy//Downloads//Desktop//Documents//flanders.txt)NameError: name flanders_filename is not defined>>> flander(Location) =(C://Users//Sandy//Downloads//Desktop//Documents//flanders.txt)SyntaxError: cant assign to function call>>> flanders_txt(location) = (C://Users//Sandy//Downloads//Desktop//Documents//flanders.txt)SyntaxError: cant assign to function call>>> flanders.txt(C://Users//Sandy//Downloads//Desktop//Documents//flanders.txt)= open(flanders.txt)SyntaxError: cant assign to function call>>> (Location) = (C://Users//Sandy//Downloads//Desktop//Documents//flanders.txt)SyntaxError: cant assign to literal>>> open(C://Users//Sandy//Downloads//Desktop//Documents//flanders.txt r)<_io.TextIOWrapper name=C://Users//Sandy//Downloads//Desktop//Documents//flanders.txt mode=r encoding=cp1252>>>> >>> flanders_filename = (C://Users//Sandy//Downloads//Desktop//Document//flanders.txt)>>> open(flanders_filename r)Traceback (most recent call last):  File <pyshell#11> line 1 in <module>    open(flanders_filename r)IOError: [Errno 2] No such file or directory: C://Users//Sandy//Downloads//Desktop//Document//flanders.txt>>> <_io.TextIOWrapper name=C://Users//Sandy//Downloads//Desktop//Documents//flanders.txt mode=r encoding=cp1252>.readline()SyntaxError: invalid syntax>>> C://Users//Sandy//Downloads//Desktop//Documents//flanders.txt, Unresolved,0
Thursday September 26th 2013 5:24 pm -04:00,Thank you so much. I used yours and other suggestions and the only problem I have now is with the title which says this.ï»¿In Flanders FieldsHere is a link to all the examples in my dropbox. https://www.dropbox.com/s/1mqwkq2w95j1n3t/PrintnotesFlandersmyfile.py With a new text file of the document I did this new one and it comes out right. Thanks for your help. https://www.dropbox.com/s/r5l7bjfa6o2cxqk/inflandersPrintNotes.py, Unresolved,0
Thursday September 26th 2013 12:02 am -04:00,I have been working for hours trying to get the file on my vista. It doesnt look promising. Im getting a unicode error. Python 3.2.3 (default Apr 11 2012 07:15:24) [MSC v.1500 32 bit (Intel)] on win32Type copyright credits or license() for more information.>>> flanders_filename = C:\Users\Sandy\Downloads\Desktop\DocumentsInFlandersFields1.odtSyntaxError: (unicode error) unicodeescape codec cant decode bytes in position 2-4: truncated \UXXXXXXXX escape>>> flanders_filename = \Users\Sandy\Downloads\Desktop\DocumentsInFlandersFields1.odtSyntaxError: (unicode error) unicodeescape codec cant decode bytes in position 0-2: truncated \UXXXXXXXX escape>>> flanders_filename =  \Users\Sandy\Downloads\Desktop\Documents\InFlandersFields1.odt SyntaxError: (unicode error) unicodeescape codec cant decode bytes in position 1-3: truncated \UXXXXXXXX escape>>> flanders_filename =  C:\Users\Sandy\Downloads\Desktop\Documents\InFlandersFields1.odt SyntaxError: (unicode error) unicodeescape codec cant decode bytes in position 3-5: truncated \UXXXXXXXX escape>>> flanders_filename = \\Users\\Sandy\\Downloads\\Desktop\\Documents\\InFlandersFields1.odt>>> flanders_file = open(flanders_filename r)>>> flanders_file.readline()Traceback (most recent call last):  File <pyshell#6> line 1 in <module>    flanders_file.readline()  File C:\Python32\lib\encodings\cp1252.py line 23 in decode    return codecs.charmap_decode(inputself.errorsdecoding_table)[0]UnicodeDecodeError: charmap codec cant decode byte 0x81 in position 710: character maps to <undefined>>>>  Does anyone have any ideas?  Thanks, Unresolved,0
Thursday September 26th 2013 12:27 pm -04:00,Unfortunately I am still trying. I saved it to UTF - 8 but python or I am just not seeing it. I appreciate your suggestions and will keep trying. , Unresolved,0
Thursday September 26th 2013 9:51 pm -04:00,I have tried this question every way possible and dont have a clue how to do a file we dont have in our system. This question says.this is a plain text file. So I have tried to put cat.text in a number of ways. Here is the latest. Anyone have a suggestion? ===========>>> cat.txt = cat_filenameSyntaxError: cant assign to literal>>> cat_filename = cat.txt>>> cat_filename = KeyboardInterrupt>>> >>>  cat_filename = (the cat  in the hat) SyntaxError: unexpected indent>>> >>> cat_filename = (the cat  in the hat)>>> open(cat_filename r)Traceback (most recent call last):  File <pyshell#125> line 1 in <module>    open(cat_filename r)TypeError: invalid file: (the cat in the hat)>>> , Resolved,0
Friday September 27th 2013 1:47 am -04:00,Thanks Nick. Thats a big help. I wish I had that before I took the test. , Resolved,0
Friday September 27th 2013 11:17 am -04:00,You need to start a new post. Board 1 txt is crashing my google. I will start a post and you can add itHere is the new post. https://class.coursera.org/programming1-002/forum/thread?thread_id=2705, Resolved,0
Wednesday September 25th 2013 2:04 pm -04:00,I have rewritten this program several times and checked and double checked it with week 6 video 3 instructions. For some reason I am getting 40.0 instead of 85.0. Can anyone see what I am doing wrong? Thanks so much. Here is the code. def calculate_average(asn_grades):    (list of list of [str number]) -> float    Return the average of the grades in asn_grades.    >>> calculate_average([[A1 80] [A2 90]])    85.0        total = 0    for item in asn_grades:          total = total + item[1]     return total  /  len(asn_grades)    After I run this program when I call the example in the docstring I get this:>>> >>> calculate_average([[A1 80] [A2 90]])40.0        , Resolved,0
Wednesday September 25th 2013 2:37 pm -04:00,Thanks for responding Tom. I restarted -- is that what you mean? And I am only trying the example in the doc string....., Resolved,0
Wednesday September 25th 2013 3:15 pm -04:00,My python had a major melt down yesterday. Maybe it has had an effect on something. Thanks for looking at that Tom. I guess I will be averaging the old fashioned way. , Resolved,0
Wednesday September 25th 2013 3:23 pm -04:00,Tom I just redid the function by copying it from my own files here like you did and it worked! Very strange solution but Ill take it. Thanks again. , Resolved,0
Friday September 27th 2013 4:26 pm -04:00,After all the work of putting InFlanders Fields into python it now says it is not defined. I am getting errors so I cant test the questions. I know its there. Ive called it by the path name everything I can think of. I would like to know how to test it. Any other suggestions as I tried everything here. Thanks. , Unresolved,3
Saturday September 28th 2013 2:09 am -04:00,I cant even get step two. Ive been trying all day. I really dont even understand it. Its very frustrating. If this is beginner I guess I should do something else. , Unresolved,0
Saturday September 28th 2013 3:35 am -04:00,Youre very kind Tom. My loops just dont work. First I get a letter than errors. Im not even sure what its asking for. At least I have 5 points. , Unresolved,1
Saturday September 28th 2013 6:11 am -04:00,Hi Tom I was doing OK with your example until I got this. [M T W T F S S][Monday TuesdayWednesdayThursday Friday Saturday Sunday]Traceback (most recent call last):  File <pyshell#70> line 1 in <module>    [M T W T F S S][Monday TuesdayWednesdayThursday Friday Saturday Sunday]TypeError: list indices must be integers not tuple>>> , Unresolved,0
Thursday September 26th 2013 1:32 pm -04:00,The problem summary:I have tried all night and one whole day to upload the file to my windows vista.  I dont usually give up but I dont have unlimited time. Is this necessary to pass this course? Steps to reproduce:Here are the last few attempts. Python 3.2.3 (default Apr 11 2012 07:15:24) [MSC v.1500 32 bit (Intel)] on win32Type copyright credits or license() for more information.>>> flanders Location = (C:\Users\Sandy\Downloads\Desktop\Documents)SyntaxError: invalid syntax>>> flanders_filename = (C:\Users\Sandy\Downloads\Desktop\Documents\flanders.txt)SyntaxError: (unicode error) unicodeescape codec cant decode bytes in position 2-4: truncated \UXXXXXXXX escape>>> flanders_filename-(C://Users//Sandy//Downloads//Desktop//Documents//flanders.txt)Traceback (most recent call last):  File <pyshell#3> line 1 in <module>    flanders_filename-(C://Users//Sandy//Downloads//Desktop//Documents//flanders.txt)NameError: name flanders_filename is not defined>>> flander(Location) =(C://Users//Sandy//Downloads//Desktop//Documents//flanders.txt)SyntaxError: cant assign to function call>>> flanders_txt(location) = (C://Users//Sandy//Downloads//Desktop//Documents//flanders.txt)SyntaxError: cant assign to function call>>> flanders.txt(C://Users//Sandy//Downloads//Desktop//Documents//flanders.txt)= open(flanders.txt)SyntaxError: cant assign to function call>>> (Location) = (C://Users//Sandy//Downloads//Desktop//Documents//flanders.txt)SyntaxError: cant assign to literal>>> open(C://Users//Sandy//Downloads//Desktop//Documents//flanders.txt r)<_io.TextIOWrapper name=C://Users//Sandy//Downloads//Desktop//Documents//flanders.txt mode=r encoding=cp1252>>>> >>> flanders_filename = (C://Users//Sandy//Downloads//Desktop//Document//flanders.txt)>>> open(flanders_filename r)Traceback (most recent call last):  File <pyshell#11> line 1 in <module>    open(flanders_filename r)IOError: [Errno 2] No such file or directory: C://Users//Sandy//Downloads//Desktop//Document//flanders.txt>>> <_io.TextIOWrapper name=C://Users//Sandy//Downloads//Desktop//Documents//flanders.txt mode=r encoding=cp1252>.readline()SyntaxError: invalid syntax>>> C://Users//Sandy//Downloads//Desktop//Documents//flanders.txtScreenshot:, Resolved,0
Thursday September 26th 2013 1:57 pm -04:00,Thanks so much for responding. This is what I have now. >>> C://Users//Sandy//Downloads//Desktop//Documents//flanders.txtC://Users//Sandy//Downloads//Desktop//Documents//flanders.txt>>> myFile = open(C://Users//Sandy//Downloads//Desktop//Documents//flanders.txt r)flanders_filename-(C://Users//Sandy//Downloads//Desktop//Documents//flanders.txt)>>> flanders_file.readline()Traceback (most recent call last):  File <pyshell#15> line 1 in <module>    flanders_file.readline()NameError: name flanders_file is not defined>>> , Resolved,0
Thursday September 26th 2013 2:08 pm -04:00,I just got the first line. Thanks so much. >>> myFile.readline()ï»¿In Flanders Fields, Resolved,0
Thursday September 26th 2013 5:20 pm -04:00,Thanks Anon and everyone else for your help. The only problem I have is in the title as you can see which prints this: ï»¿In Flanders FieldsI have all the examples here on a link to my dropbox if it might help someone elsehttps://www.dropbox.com/s/1mqwkq2w95j1n3t/PrintnotesFlandersmyfile.py  , Resolved,0
Thursday September 26th 2013 6:01 pm -04:00,This is just a file from notepad. I dont have the details for it except that I usted the UTP = 8 format as you suggested. https://www.dropbox.com/s/0r4gzz0hesxy8qs/flanders.txt, Resolved,0
Thursday September 26th 2013 8:21 pm -04:00,https://www.dropbox.com/s/r5l7bjfa6o2cxqk/inflandersPrintNotes.py Thanks Tom. Its taken a while but I think I fixed it. Thanks so much for all your help. , Resolved,0
Saturday September 28th 2013 12:30 pm -04:00,Thanks for all the help. , Resolved,0
Friday September 27th 2013 10:43 pm -04:00,Ive been staring at it all day and getting nowhere. I think its too much for a beginner -- well at least for me. , Unresolved,0
Friday September 27th 2013 11:23 am -04:00,I cannot copy it on to my notepad. I was able to copy the word list but the board list is crashing my google. there is a similar post that I will try to link to. When I try to copy and paste this list google closes up. I havent even tried to get the rest. Thanks.(I have windows vista)., Resolved,0
Friday September 27th 2013 12:11 pm -04:00,Thats great. How do you create a txtor notepad? Thanks, Resolved,0
Friday September 27th 2013 12:55 pm -04:00,My problem is that the notepad will not copy the board1.txt document. Are you saying that it can copy it in Python? (Sorry but I am new to programming and had a late start with computers)., Resolved,0
Friday September 27th 2013 4:47 pm -04:00,Im really sorry but even going near boardlist1 crashes the google. I dont know why. I cant figure out where you are telling me to add the text document in Python. I only have the command line the idle GUI and the Python Shell and Window and I dont find anything there that deals with putting in a text document. I spent three days putting in In Flanders Fields so I think I could put it in from the notepad if  could get it. Thanks for your help. , Resolved,0
Friday September 27th 2013 3:15 pm -04:00,I just tried again and google crashed. I cannot find txtor on python or any information on search. I will try to copy it myself on to notepad I have poor eyesight. I will do my best. , Resolved,0
Saturday September 28th 2013 12:29 pm -04:00,I copied it and emailed it to myself from another computer. I hope that works but I would be curious to find out how this method worked. , Resolved,0
Saturday September 28th 2013 8:49 pm -04:00,I have only one of these right the first one. So thats what I am working on. I called range on the board and got the right answer in the funtion. Now Im trying to work it out. , Unresolved,0
Saturday September 28th 2013 9:40 pm -04:00,The nested loops tape is not working.. I guess Ill be here a while. , Unresolved,0
Saturday September 28th 2013 9:59 pm -04:00,This student has all the pdfs on his google drive. If you have it you can add them to yours. Stijn Blommerde·, Unresolved,0
Saturday September 28th 2013 10:10 pm -04:00,Hi Tom Im trying to do this function. I only got the first one right.make_str_from_column([[A N T T] [X S O B]] 1)Traceback (most recent call last):  File <pyshell#8> line 1 in <module>    make_str_from_column([[A N T T] [X S O B]] 1)  File C:\Users\Sandy\Downloads\a3.py line 77 in make_str_from_column    for  row in (range(len(0))):TypeError: object of type int has no len(), Unresolved,0
Saturday September 28th 2013 10:13 pm -04:00,I guess Ill have to do a loop. I tried not to as I am not very good at them. I did len on the board and it worked for the example. Now I have this. , Unresolved,0
Saturday September 28th 2013 10:30 pm -04:00,Ok So aList = [hello this is a list]for word in list:print (word)hellothisisalist  but what is my variable? O K so Ive got for char in a list. For char in list1 print :something in list 1For char in list2 print algo in list 2So if my list is( (A B C D)(EFGH)) is that one list or two? Do I have to split them up or can I call them list 1 and list 2?, Unresolved,0
Saturday September 28th 2013 11:16 pm -04:00,Do I need to index or slice it? Im trying all different ways. I need to get NS. Can I just list them?, Unresolved,0
Saturday September 28th 2013 11:49 pm -04:00,Its not giving me the right out put. I am not getting an error but no words.I just looked again. I am getting  the whole string.([[A N T T] [X S O B]] 1) No I was wrong. I have an error. raceback (most recent call last):  File <pyshell#17> line 1 in <module>    make_str_from_column([[A N T T] [X S O B]] 1)  File C:\Users\Sandy\Downloads\a3.py line 76 in make_str_from_column    for word in list:TypeError: type object is not iterabl, Unresolved,0
Sunday September 29th 2013 12:00 am -04:00,I fixed that one I hope. Now I have this one. str_from_column([[A N T T] [X S O B]] 1)Traceback (most recent call last):  File <pyshell#25> line 1 in <module>    make_str_from_column([[A N T T] [X S O B]] 1)NameError: name make_str_from_column is not defined>>> V, Unresolved,0
Sunday September 29th 2013 12:05 am -04:00,I think I fixed that but I have the same error back.make_str_from_column([[A N T T] [X S O B]] 1)Traceback (most recent call last):  File <pyshell#26> line 1 in <module>    make_str_from_column([[A N T T] [X S O B]] 1)  File C:\Users\Sandy\Downloads\a3.py line 76 in make_str_from_column    for word in (list):TypeError: type object is not iterable, Unresolved,0
Sunday September 29th 2013 12:18 am -04:00,Thanks Tom but I must have done something wrong. I had a runtime error. )  File C:\Users\Sandy\Downloads\a3.py line 84 in make_str_from_column    return make_str_from_column(board column_index)  File C:\Users\Sandy\Downloads\a3.py line 84 in make_str_from_column    return make_str_from_column(board column_index)  File C:\Users\Sandy\Downloads\a3.py line 84 in make_str_from_column    return make_str_from_column(board column_index)RuntimeError: maximum recursion depth exceeded>>> , Unresolved,0
Sunday September 29th 2013 1:14 am -04:00,Well at tleast I dont have errors but Im getting this. make_str_from_column([[A N T T] [X S O B]] 1  XSOB, Unresolved,0
Sunday September 29th 2013 1:22 am -04:00,Now I have this. Traceback (most recent call last):  File <pyshell#35> line 1 in <module>    make_str_from_column([[A N T T] [X S O B]] 1)  File C:\Users\Sandy\Downloads\a3.py line 89 in make_str_from_column    return .join(b)TypeError: sequence item 0: expected str instance list found>>> , Unresolved,0
Sunday September 29th 2013 1:24 am -04:00,I guess I am not understanding this. We turn the lists into columns right? But we still have that list left. Ill forget about the list I guess. , Unresolved,0
Sunday September 29th 2013 1:49 am -04:00,It doesnt look anything like that. Its a mess. I have loops all over the place. Im sorry I just dont get this. You are a very good teacher and explain well. I am just having a hard time with this. There is a list inside a list. How do I get to it? I used the index. But my python just shows that I got the first list and its not even the right one. Something is wrong with my thinking. , Unresolved,0
Sunday September 29th 2013 4:47 am -04:00,Ill be working on this tomorrow as it didnt pass. Ill see if I feel like continuing but thanks for the help!., Unresolved,0
Saturday September 28th 2013 1:16 am -04:00,I just got this error and my idle isnt working. What does this mean and what can I do about it. Thanks. , Resolved,0
Saturday September 28th 2013 12:27 pm -04:00,It seems to be O K today. Im resolving the thread., Resolved,0
Saturday September 28th 2013 7:50 pm -04:00,I only passed one of board contains work in column (it passed the expected tests). I I should start over I guess. I think I need help with a loop.I tried a few and will keep trying. On the other hand I did call Len on the board but I am not sure how to procede.I tried it the other way and was out of range. Its been a long day. , Unresolved,0
Saturday September 28th 2013 8:19 pm -04:00,Thanks Lillian. It took me three days to get this far. I used for index in range(len(G)) and G was the board:  and I dont really know how to do psuedo code but thats the gist of it. I see by Croitorus post that that will only give me the one correct value on the test case but Im not really sure what to do. .Thanks for responding. , Unresolved,0
Saturday September 28th 2013 8:46 pm -04:00,Thanks I am in the wrong place. I will need this later. I still have to get the one before. Thanks. , Unresolved,0
Saturday September 28th 2013 8:54 pm -04:00,Ill do this now as I have no idea about the other on. Thanks so much Lillian!. , Unresolved,0
Saturday September 28th 2013 9:17 pm -04:00,I see Im going to have to get that column function first. Thanks a lot. , Unresolved,0
Wednesday October 2nd 2013 8:26 pm -04:00,Finally got it! Now to read boards..., Resolved,0
Monday September 30th 2013 12:33 am -04:00,Ive been working for days and nights and this is the best I can do. I really appreciate all the help that everyone has given especially Tom and many others. I really need the Python for Dummies version. I would recommend that maybe the course description be changed so a first time programmer might know a little more about what they are getting into. I may try again someday.Even though I am a bit discouraged this was a great class. Thanks so much for giving it.  , Unresolved,3
Monday September 30th 2013 12:52 am -04:00,Thanks Anon. and good luck on the assignment. , Unresolved,0
Monday September 30th 2013 1:15 am -04:00,Tom thanks so much for all your help. I really appreciate it. , Unresolved,1
Monday September 30th 2013 1:21 am -04:00,Whats the IRC Channel Tom? Im lookin it up. Is it like google chat or skype?, Unresolved,0
Monday September 30th 2013 5:42 am -04:00,Thanks everyone for your encouragement. I am doing better and even got some more right. , Unresolved,4
Tuesday October 1st 2013 1:57 pm -04:00,I guess its just reading one word. I have checked the len and it must have something to do with column index not reading more than the first word. I have checked this and it seems O K. Any other suggestions?  Python 3.2.3 (default Apr 11 2012 07:15:24) [MSC v.1500 32 bit (Intel)] on win32Type copyright credits or license() for more information.>>> ================================ RESTART ================================>>> >>>  num_words_on_board([[A N T T] [X S O B]] [ANT BOX SOB TO])1>>> num_words_on_board([[A N T T] [X S O B]] [ANT BOX SOB])1>>> num_words_on_board([[A N T T] [X S O B]] [ANT BOX TO])1>>> num_words_on_board([[A N T T] [X S O B]] [ANT])1>>>  num_words_on_board([[A N T T] [X S O B]] ), Unresolved,0
Tuesday October 1st 2013 2:24 pm -04:00,Thanks for responding Eric. I have that line on the top of the loop. Is that right?  Also here are the tests for make string and word in a row. >>> make_str_from_row([[A N T T] [X S O B]] 0)ANTT>>> make_str_from_row([[A N T T] [X S O B]] 1)XSOB>>> make_str_from_column([[A N T T] [X S O B]] 1)NS>>> make_str_from_column([[A N T T] [X S O B]] 2)TO>>> make_str_from_column([[A N T T] [X S O B]] 3)TB>>> make_str_from_column([[A N T T] [X S O B]] 0)AX>>> , Unresolved,0
Tuesday October 1st 2013 2:34 pm -04:00,Thanks Tom. That is exactly what I have. temp = 0for item in list:        if function ():            temp = temp + 1return temp, Unresolved,0
Tuesday October 1st 2013 2:41 pm -04:00,Here is a problem.board_contains_word_in_row([[A N T T] [X S O B]] SOB)>>>FALSE, Unresolved,0
Tuesday October 1st 2013 2:54 pm -04:00,I just took out the return False in board_contains_word_in_row and then board_contains_word_in_row([[A N T T] [X S O B]] SOB)>>>TRUE but if I put it back in I get board_contains_word_in_row([[A N T T] [X S O B]] SOB)>>>FALSE.Do we need the return FALSE there?, Unresolved,0
Tuesday October 1st 2013 3:11 pm -04:00,When I put it back on I get board_contains_word_in_row([[A N T T] [X S O B]] SOB)    FALSE, Unresolved,0
Tuesday October 1st 2013 3:27 pm -04:00,Hi Owain Look at Toms template above. He has a variable name not a function name on the top line. And the second line needs the same change. whoops that looks OK. and also variable = variable + 1return variable.Change those and you should be ok. ( I am just a beginner but give it a try.)temp = 0for item in list:        if function ():            temp = temp + 1return temp, Unresolved,0
Tuesday October 1st 2013 4:13 pm -04:00,. Look back at Toms instructions., Unresolved,0
Tuesday October 1st 2013 3:35 pm -04:00,Hi Tom it is exactly right and still is giving:board_contains_word_in_row([[A N T T] [X S O B]] SOB)>>>FALSE, Unresolved,0
Tuesday October 1st 2013 3:44 pm -04:00,Hi Owain Top line look at the template. You need a variable not a function. Item = 0Item = item + 1You have a function there. , Unresolved,0
Tuesday October 1st 2013 3:44 pm -04:00,Hi Tom make string from  a row I have:    item =       for char in function        item = item + char             return item, Unresolved,0
Tuesday October 1st 2013 4:05 pm -04:00,I didnt change it. Anything else gives me an error such asTraceback (most recent call last):  File <pyshell#41> line 1 in <module>    board_contains_word_in_row([[A N T T] [X S O B]] SOB)  File C:\Users\Sandy\Downloads\a3.py line 101 in board_contains_word_in_row    if word in make_str_from_row(board row_index):  File C:\Users\Sandy\Downloads\a3.py line 55 in make_str_from_row    for char in board(row_index):TypeError: list object is not callable, Unresolved,0
Tuesday October 1st 2013 4:07 pm -04:00,Im sorry. I explained it wrong. I have exactly what you said. I called it the wrong thing. Sorry and thanks. I have to put it backe in. , Unresolved,0
Tuesday October 1st 2013 4:17 pm -04:00,  whoops I have functions here =     for item in board:        function = function + item              return functionThat must be it. Thanks Tom. I will update in a second. , Unresolved,0
Tuesday October 1st 2013 4:29 pm -04:00,I get an error Tom. I did not change this one. I think that was the only way that worked. )Traceback (most recent call last):  File <pyshell#49> line 1 in     make_str_from_column([[A N T T] [X S O B]] 1)NameError: name make_str_from_column is not defined, Unresolved,0
Tuesday October 1st 2013 4:36 pm -04:00,Tom I changed item to same as one in word form row and now I get this.make_str_from_column([[A N T T] [X S O B]] 1)XSOB, Unresolved,0
Tuesday October 1st 2013 4:40 pm -04:00, same item as in word for row=     for char in board:        item = item + char              return item, Unresolved,0
Tuesday October 1st 2013 4:43 pm -04:00,I have never changed this Tom. When I try an other item it throws and error.. I have a paste on my notepad of everything from 3:30 in the morning and that is what it says. I have always gotten points off on make string from a column and thats  why Im sure. , Unresolved,0
Tuesday October 1st 2013 4:55 pm -04:00,Thanks Tom. I have a new error.  make_str_from_column([[A N T T] [X S O B]] 1)Traceback (most recent call last):  File <pyshell#56> line 1 in <module>    make_str_from_column([[A N T T] [X S O B]] 1)  File C:\Users\Sandy\Downloads\a3.py line 74 in make_str_from_column    for char in board(column_index):TypeError: list object is not callable>>> make_str_from_column([[A N T T] [X S O B]] 1), Unresolved,0
Tuesday October 1st 2013 4:58 pm -04:00,Got it. I change it to just board. Now for some more testing. Thanks so much Tom. , Unresolved,0
Tuesday October 1st 2013 5:03 pm -04:00,Oh well. I still get this. num_words_on_board([[A N T T] [X S O B]] [ANT BOX SOB TO])1, Unresolved,0
Tuesday October 1st 2013 5:05 pm -04:00,For column index I haveitem(same name as row index item) : =     for char in board:        item + item + char              return item, Unresolved,0
Tuesday October 1st 2013 5:08 pm -04:00,I think I see the problem.Be right back.No everything is correct except board contains word in row is still False. board_contains_word_in_row([[A N T T] [X S O B]] SOB)Falsemake_str_from_column([[A N T T] [X S O B]] 1)NS>>> make_str_from_column([[A N T T] [X S O B]] 0)AX>>> make_str_from_column([[A N T T] [X S O B]] 2)TO>>> make_str_from_column([[A N T T] [X S O B]] 3)TB>>>>>>>>> make_str_from_row([[A N T T] [X S O B]] 0)ANTT>>> make_str_from_row([[A N T T] [X S O B]] 1)XSOB, Unresolved,0
Tuesday October 1st 2013 5:25 pm -04:00,Ok Ill give it a try. Thanks. , Unresolved,0
Tuesday October 1st 2013 5:36 pm -04:00,Tom I am really sorry that I am not understanding it. This is my error. >>> make_str_from_column([[A N T T] [X S O B]] 1)Traceback (most recent call last):  File <pyshell#79> line 1 in <module>    make_str_from_column([[A N T T] [X S O B]] 1)  File C:\Users\Sandy\Downloads\a3.py line 75 in make_str_from_column    word = word + char[column_index]TypeError: unsupported operand type(s) for +: int and str>>> , Unresolved,0
Tuesday October 1st 2013 5:53 pm -04:00,Yes but it doesnt matter because now Im getting this for make string from columnColumn function item =     item= 0    for item in board:        item = item+ char              return itemI am getting this error. make_str_from_column([[A N T T] [X S O B]] 3)Traceback (most recent call last):  File <pyshell#88> line 1 in     make_str_from_column([[A N T T] [X S O B]] 3)  File C:\Users\Sandy\Downloads\a3.py line 76 in make_str_from_column    word = word + char[column_index]NameError: global name char is not defined, Unresolved,0
Tuesday October 1st 2013 6:04 pm -04:00,I think I see 2 return statements but I still get the error about char not defined. make_str_from_column([[A N T T] [X S O B]] 3)Traceback (most recent call last):  File <pyshell#89> line 1 in <module>    make_str_from_column([[A N T T] [X S O B]] 3)  File C:\Users\Sandy\Downloads\a3.py line 76 in make_str_from_column    word = word + char[column_index]NameError: global name char is not d, Unresolved,0
Tuesday October 1st 2013 6:08 pm -04:00,def make_str_from_column(board column_index):     (list of list of str int) -> sIrIt wants a string right and I need an integer. return item char??, Unresolved,0
Tuesday October 1st 2013 6:22 pm -04:00,Tom I used the same item as in row. I think I have to change it. Am I right?, Unresolved,0
Tuesday October 1st 2013 6:35 pm -04:00,I am really  sorry that I cant get beyond this. make_str_from_column([[A N T T] [X S O B]] 3)make_str_from_column([[A N T T] [X S O B]] 3[X S O B 3][X S O B 3That doesnt seem right, Unresolved,0
Tuesday October 1st 2013 7:19 pm -04:00,Ok so you are saying I should change the wording. I am not using those words. Will do.Thanks. , Unresolved,0
Tuesday October 1st 2013 11:51 pm -04:00,The problem summary:For Assignment 3 when I tried to get the txt files my google crashed and I was unable to copy and paste them. I tried another browser and emailing and even though I used UTF - 8. I have an extra character. Steps to reproduce:This is what the first word looks like in Wordlist 1.ï»¿CRUNCHYThe same thing happened with In Flanders Fields but I got a clean copy from the cloud. Thanks for your assistance. Screenshot:This is a link of what happened with In Flanders Fields.https://www.dropbox.com/s/1mqwkq2w95j1n3t/PrintnotesFlandersmyfile.py, Unresolved,0
Wednesday October 2nd 2013 1:25 am -04:00,Thanks so much Tom. You are always here for us. , Unresolved,0
Wednesday October 2nd 2013 7:24 pm -04:00,I am having trouble opening the file in idle. Ive tried useing the file path and the file name and both but nothing happens. Do we have to do something else?  Thanks Whoops this is read file. I think I am in the wrong place. , Resolved,0
Wednesday October 2nd 2013 7:58 pm -04:00,Thank for responding.  I can open it with the path name in idle but it just wont work in a function call. , Resolved,0
Wednesday October 2nd 2013 8:41 pm -04:00,Hi Tom! Thanks for answering. Do you mean like copy and paste? I know they are two lists can their contents be know without a path? , Resolved,0
Wednesday October 2nd 2013 9:48 pm -04:00,Tom you have done so much for us why dont you get some rest. Im probably not going to get these last two and its O K. , Resolved,0
Wednesday October 2nd 2013 10:10 pm -04:00,I still have to do read files. My computer was burning up because it was on for so many days and nights and I had to give it a rest. I tried all last night to do read.  I have no idea how to put apy file and a .txt together. I can hardly find my files when I put them in the computer. Its amazing that I got this far( with a lot of help) but I did learn a lot. Ill take a look at this thread but its the board thread. Thanks again for all your help. , Resolved,0
Wednesday October 2nd 2013 10:32 pm -04:00,Thanks Anne. I have a file with the two list text called MyFile but do I need to call it a py file because it wont go into the idle. , Resolved,0
Wednesday October 2nd 2013 10:57 pm -04:00,Thanks Anne. I made the file  but I dont get the rest of what you said. The py files are in the idle and how can I put text files in there??, Resolved,0
Wednesday October 2nd 2013 10:54 pm -04:00,I was in string for row and column for days and finally got it. Try to post psuedo code. I dont know If I can help but maybe. , Resolved,0
Wednesday October 2nd 2013 11:20 pm -04:00,Well I guess I was not successful in loading the files. Traceback (most recent call last):  File <pyshell#0> line 1 in <module>    file = open(board1.txt r)IOError: [Errno 2] No such file or directory: board1.txt>>> Traceback (most recent call last):  File <pyshell#0> line 1 in <module>    file = open(board1.txt r)IOError: [Errno 2] No such file or directory: board1.txt>>> , Resolved,0
Wednesday October 2nd 2013 11:33 pm -04:00,No Im on a windows vista. The py. files are in a big console like a box. You can see the files all together but I cant seperate the py files out or can I ??, Resolved,0
Wednesday October 2nd 2013 11:46 pm -04:00,Yes I do.. I will look again. Thanks Anne. , Resolved,0
Wednesday October 2nd 2013 11:36 pm -04:00,I did it a little different.  thing_you said  =     for row in board:        thing you said = thing you said + row[thingyousaid]              return thing you said     , Resolved,0
Wednesday October 2nd 2013 11:45 pm -04:00,Give mine a try. It works. , Resolved,0
Thursday October 3rd 2013 12:06 am -04:00,Anon I had that same problem. So mine worked after and I fixed num of words of board .It was specified that column index len was 0 in the doctstring and a number of posts stated row  len. . My problem was indentation with the true false in the next two functions. You have to have the row index and column index right or everything is wrong. And some how my indents had changed . There is starter code given by the doc string at board contains word from a column and you cant change it a bit. , Resolved,0
Wednesday October 2nd 2013 11:54 pm -04:00,I am really embarrassed to say that I dont know what a file manager is or if I even have one. I dont know much about computers (obviously)., Resolved,0
Thursday October 3rd 2013 12:10 am -04:00,Thanks I know where that is I think. , Resolved,0
Thursday October 3rd 2013 12:40 am -04:00,Congrats Anon! Good job., Resolved,0
Thursday October 3rd 2013 1:16 am -04:00,I did get them in the same folder with Idle. Now I have this error.  wordlist.txt.readline()Traceback (most recent call last):  File <pyshell#1> line 1 in <module>    wordlist.txt.readline()NameError: name wordlist is not defined>>> wordlist.txt = words_fileTraceback (most recent call last):  File <pyshell#2> line 1 in <module>    wordlist.txt = words_fileNameError: name words_file is not defined>>> , Resolved,0
Thursday October 3rd 2013 2:37 am -04:00,Idle is just not picking up the two othe files in the folder. I can only open the python folder with the idle. The program doesnt see the others. Doing the paths didnt work. I guess I better go study. Thanks so much everyone. , Resolved,1
Thursday October 3rd 2013 3:45 am -04:00,Thanks Yurily. I really appreciate your response.  So the answer is none. So I am off. I still need read. Thanks a lot. , Resolved,0
Friday October 4th 2013 3:56 am -04:00,Whatever I do I cant get my wordlists to open. I have read every post and have worked for hours on this problem. I dont know how to get the wordlist . Can anyone help? Thanks>>> open(wordlist1.txt)<_io.TextIOWrapper name=wordlist1.txt mode=r encoding=cp1252>>>> wordlist1.txt.readline()Traceback (most recent call last):  File <pyshell#14> line 1 in <module>    wordlist1.txt.readline()NameError: name wordlist1 is not defined    name = wordlist1.txtNameError: name wordlist1 is not defined>>> name = wordlist1.txt>>> wordlist1.txt = wordlist1_wordfileSyntaxError: cant assign to literal>>> wordlist1.txt.read()Traceback (most recent call last):  File <pyshell#26> line 1 in <module>    wordlist1.txt.read()AttributeError: str object has no attribute read>>> , Resolved,0
Friday October 4th 2013 4:16 am -04:00,Thanks very much. Now I can see the list!, Resolved,0
Friday October 4th 2013 4:17 am -04:00,Thanks a lot. I think it will. , Resolved,0
Friday October 4th 2013 4:21 am -04:00,So thats great. How do I get to the next word?>>> file = file.readline()>>> lineCRUNCHY>>> lineCRUNCHY>>> lineCRUNCHY>>> , Resolved,0
Friday October 4th 2013 6:25 am -04:00,Thanks very much. That helped a lot.  open(wordlist1.txt)<_io.TextIOWrapper name=wordlist1.txt mode=r encoding=cp1252>>>> file = open(wordlist1.txt)>>> >>> >>> line = file.readline()>>> >>> lineCRUNCHY>>> line = file.readline()>>> lineCOWS>>> >>> line = file.readline()>>> lineEAT>>> line = file.readline()>>> lineGRASS>>> line, Resolved,0
Friday October 4th 2013 8:31 am -04:00,Its great information. Thank you very much. , Resolved,0
Friday October 4th 2013 8:34 am -04:00,Thank you., Resolved,0
Friday October 4th 2013 9:33 am -04:00,I am just testing but I cant figure out how to get words_files into my code with this text wrapper. Ive tried many different ways but always get an error code.  def read_words(words_file):     (file open for reading) -> list of str    Return a list of all words (with newlines removed) from open file    words_file.    Precondition: Each line of the file contains a word in uppercase characters    from the standard English alphabet.        open(wordlist1.txt r)    >>> >>> >>> open(wordlist1.txt r)<_io.TextIOWrapper name=wordlist1.txt mode=r encoding=cp1252>>>> file = open(wordlist1.txt)>>> line = words_file.readline()Traceback (most recent call last):  File <pyshell#42> line 1 in <module>    line = words_file.readline()NameError: name words_file is not defined def read_words(words_file):     (file open for reading) -> list of str   I know that the file is already open for reading so I dont  open it. So if I proceed with my file open it looks like this. open(wordlist1.txt r)<_io.TextIOWrapper name=wordlist1.txt mode=r encoding=cp1252>>>> file = open(wordlist1.txt)Do I call that a words_file?, Resolved,0
Friday October 4th 2013 9:53 am -04:00,I just got it. Thanks for the help. Now to read board. , Resolved,0
Saturday October 5th 2013 4:11 pm -04:00,Samuel I also  had that problem.There is a whole post on that problem. For me my indentations on my True and False statements were not quite correct. https://class.coursera.org/programming1-002/forum/thread?thread_id=3025 , Unresolved,0
Saturday October 5th 2013 4:07 pm -04:00,Thanks so much Tom. I finally  did it!. Your patient explanations and teaching moments are deeply appreciated. Take care. , Unresolved,0
